06-release-engineering-analytics.Rmd

```diff
@@ -424,11 +424,91 @@ but that is probably still part of RRs (even though it's so specific) [@castellu
 
 ### RQ3: Future Research
 
-This section will give an answer to the question:
+This section aims to answer the question:
 _What aspects of modern release engineering make for
 relevant study objects in future software analytics research?_
 
-<!-- TODO -->
+#### General
+
+The body of literature that we analyzed for this survey mostly comprised case
+studies that employed quantitative analysis methods.
+From these studies, interesting conclusions have been drawn about effects of
+release engineering practices on software development processes in specific
+contexts.
+However, the generalizability of the findings in these case studies is very
+limited.
+Therefore, in general many studies suggest that future research efforts focus on
+performing additional case studies, both to verify existing findings and to
+study new relationships and new contexts @karvonen2017a @teixeira2017a
+@khomh2015a @claes2017a @laukkanen2018a @adams2016a @castelluccio2017a.
```

Comment:
When citing papers like this, you should use this Rmd syntax:

`[@karvonen2017a; @teixeira2017a; ...; @castelluccio2017a].`

(same goes for other places in this chapter)

When literally using the author's name(s) in a sentence, the square brackets are not needed (e.g.: `@laukkanen2017a identified 40 problems` is correct)

Reply:
Thanks

Reply:
Done